keras/backend/cntk_backend.py

```diff
@@ -1072,9 +1087,11 @@ def _static_rnn(step_function, inputs, initial_states,
 
     # if the second axis is static axis, CNTK will do unroll by default
     if shape[1] is None:
-        raise ValueError('Static rnn in cntk '
-                         'could only be executed with '
-                         'second axis as static axis')
+        raise ValueError('CNTK Backend: the input of static rnn'
+                         'is with shape \'%s\', the second axis'
+                         'is not static axis. If you want to run'
+                         'rnn with non-static axis, plesae try'
+                         'dynamic rnn with sequence axis.' % shape)
```

Comment:
Need spaces at the end of every line in this message (this is also true of a few other messages)